src/steamship/data/package/package_instance.py

```diff
@@ -52,6 +65,7 @@ class PackageInstance(CamelModel):
     workspace_id: str = None
     workspace_handle: str = None
     init_status: Optional[InvocableInitStatus] = None
+    created_at: Optional[datetime] = None
```

Comment:
For instances created before this change, is this just expected to be None? I don't see where this hydration happens otherwise.

Reply:
When hydrating in from_dict, Pydantic will ignore fields that it doesn't have a slot for.  The createdAt field has been passed by the engine, but ignored by Pydantic / this object until this field is added.